scripts/updateHfDataset.py

```diff
@@ -58,10 +58,10 @@ def hfUpload():
     '''
     Upload the clean dataset to huggingface.co
     '''
-    local_filename = '2024-clean.csv'
-    dest_filename = '2024.csv'
-    username = '311-data'
-    repo_name = '2024'
+    local_filename = '2024-clean.parquet'
+    dest_filename = '2024.parquet'
+    username = 'edwinjue'
+    repo_name = '311-data-2024'
```

Comment:
this should just be `repo_name = '2024'` (should reflect https://huggingface.co/datasets/311-data/2024)